spec/models/user_spec.rb

```diff
@@ -26,5 +26,30 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # 正常系
+  describe 'positive for signup' do
+    # ユーザ名・メールアドレス・パスワードが存在すれば有効であること
+    it 'is valid with a name, an email and password' do
+      user = FactoryBot.build(:user)
```

Comment:
テストケースは`ユーザ名・メールアドレス・パスワードが存在すれば有効であること`で
factoryが`FactoryBot.build(:user)`でbuild時に値を何も渡さずfactoryのデフォルト値に依存してしまうのはよくないです。
なぜかというとfactoryの定義を変えるとテストが落ちてしまうからです。
`ユーザ名・メールアドレス・パスワードが存在すれば有効であること`の場合、
正しいユーザー名、メールアドレス、パスワードをこのテストケース内のfactoryに定義してあげるのがよいです。
以下のようなイメージです。

```suggestion
      user = FactoryBot.build(:user, email: "user@example.com", password: "password")
```